pkg/cmd/cli/cli.go

```diff
@@ -150,6 +151,25 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	return cmds
 }
 
+// changeSharedFlagDefaults changes values of shared flags that we disagree with.  This can't be done in godep code because
+// that would change behavior in our `kubectl` symlink. Defend each change.
+// 1. show-all - the most interesting pods are terminated/failed pods.  We don't want to exclude them from printing
+func changeSharedFlagDefaults(rootCmd *cobra.Command) {
+	cmds := []*cobra.Command{rootCmd}
+
+	for i := 0; i < len(cmds); i++ {
+		currCmd := cmds[i]
+		cmds = append(cmds, currCmd.Commands()...)
```

Comment:
> Habit. I trust iterative solutions more than recursive ones.

That's why I don't feel strongly. In the one side you have recursion, in the other side you edit the slice while iterating:)